libs/atqr/domain/src/lib/domain-spec/stories/showPopularGoals.disl

```diff
@@ -0,0 +1,18 @@
+domain ATQR
+
+context Challenges
+
+  showPopularGoals? :qty
+    rules:
+      - Challenges have to exist in the database
+
+    //call a function that will return a array with arrays of goals that are the same
+    groupByGoals!
+    show Goals :qty in descending order by lenght of array
```

Comment:
you should "return" and not "show". Also, you need to find, count the challenges, and order by number of challenges